### DISCUSSION TITLE
Float UI V2 has now been officially released! 🎉🥳

### DISCUSSION THREAD

**Discussion by sidiDev** [MarsX-dev/floatui#42] [Announcements]
Today, we are super excited to announce **Float UI v2**.

Do you know that we have been working very hard to finish as much work as possible so that we can ship this version and you guys can benefit from it? Yes, my friend. Let me tell you that converting components to other frameworks and to HTML with AlpineJS is really not easy and takes time. The funny thing is, sometimes when you convert a component and finish, you notice that the style doesn't 100% match the original component.

I have some very good news for you. **The new design of the website is now open source** with the components, and there are so many cool things you can directly copy and use in your next project for free. Plus, we are working on **an advanced components Editor with AI.**

Also we are launching on Product Hunt, please support us there: [https://www.producthunt.com/posts/float-ui-v2](https://www.producthunt.com/posts/float-ui-v2)

Float UI is nothing without you ❤️

**Comment by onesamket** [comment]
It's amazing I was using float UI for a while and Iove it.

**Reply by sidiDev** [reply]
Wow, that’s cool, thank you man. and happy to hear that.

**Comment by djvamps** [comment]
float ui, para astro es una mierda, no lo recomiendo
